catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/transformer/transformers/sierra/SierraItems.scala

```diff
@@ -0,0 +1,55 @@
+package uk.ac.wellcome.transformer.transformers.sierra
+
+import com.twitter.inject.Logging
+import scala.util.{Failure, Success}
+import uk.ac.wellcome.models.{
+  IdentifierSchemes,
+  SourceIdentifier,
+  UnidentifiedItem
+}
+import uk.ac.wellcome.models.transformable.SierraTransformable
+import uk.ac.wellcome.transformer.source.SierraItemData
+import uk.ac.wellcome.utils.JsonUtil._
+
+trait SierraItems extends Logging with SierraLocation {
```

Comment:
Does this need to be a trait? Could it simply be an object?

Reply:
It’s a trait because that’s the current pattern used by SierraTransformableTransformer. They could all be objects instead.